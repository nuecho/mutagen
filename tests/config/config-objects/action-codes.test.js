cfgObjectTests("actionCodes", "action-codes-config.json");
