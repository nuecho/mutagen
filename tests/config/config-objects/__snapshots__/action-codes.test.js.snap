// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[mutagen actionCodes config import-export] should create the actionCodes: the actionCodes have been created 1`] = `
Object {
  "stderr": "",
  "stdout": "+ actioncode => name: 'cfgObjectTest', type: 'transfer', tenant: 'Environment'
Completed. 1 object(s) imported.
",
}
`;

exports[`[mutagen actionCodes config import-export] should throw exception when new actionCodes to import miss mandatory properties actionCodes: exception has been thrown 1`] = `
Object {
  "stderr": "Mandatory properties [code] not set in actioncode 'name: 'emptyCfgObjectTest', type: 'transfer', tenant: 'Environment''
Cannot import configuration: some configuration objects' mandatory properties for creation are not set.
",
  "stdout": "",
}
`;

exports[`[mutagen actionCodes config import-export] should update the actionCodes: the actionCodes have been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "~ actioncode => name: 'cfgObjectTest', type: 'transfer', tenant: 'Environment'
Completed. 1 object(s) imported.
",
}
`;
